login.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+require_once 'src/Services/DatabaseConnectionServices.php';
+require_once 'src/Models/UsersModel.php';
+require_once 'src/Services/NavBarService.php';
+
+session_start();
+
+$db = DatabaseConnectionServices::connect();
+$user = new UsersModel($db);
```

Comment:
I actually think its worth moving these two lines, $db and $user, down into the if statement where you check the form has been submitted. 

Just from an efficiency perspective, no point in getting a DB connection and a model if the form hasn't actually been submitted yet.